src/yunohost/tests/test_changeurl.py

```diff
@@ -19,10 +19,8 @@
 
 
 def setup_function(function):
-    # For some reason the nginx reload can take some time to propagate
-    time.sleep(1)
```

Comment:
I still think we might want to keep that `time.sleep(1)`, apparently that was related to nginx starting. What do you think ? :wink:

Reply:
Well it was a copy past. And it was to try to fix the authenticator. But know I've found an other way to fix that. Before this PR this line wasn't here...